highlight.io/pages/demo-confirmation.tsx

```diff
@@ -0,0 +1,23 @@
+import { FooterCallToAction } from '../components/common/CallToAction/FooterCallToAction'
+import Footer from '../components/common/Footer/Footer'
+import Navbar from '../components/common/Navbar/Navbar'
+import { Typography } from '../components/common/Typography/Typography'
+
+export default function Highlight404() {
```

Comment:
Why is this called `Highlight404`?

Reply:
oh oops, i copied the 404 page and forgot to change that